package.json

```diff
@@ -36,4 +39,4 @@
     "karma-jasmine-html-reporter": "~1.7.0",
     "typescript": "~4.4.3"
   }
-}
+}
```

Comment:
Please add an empty line to the end of file. Please make sure that this is done to all the files.

Reply:
Empty line added for every file. Can I know the reason behind having an empty line at the end of each file?

Reply:
@nihal-rg This was documented in the Git Training section [here](https://coda.io/d/_dHOXTAtFBHO/Introductory-Git_sufIs#_lugRv).
The fact that you were supposed to go over the complete Git document at the start of your training and now you are asking this question leaves me wondering if you will continue to forget concepts going forward.

Reply:
I am sorry, I do remember going through the content at the start of my training. This will not be repeated.